opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/ingestion-csv-test.ts

```diff
@@ -0,0 +1,139 @@
+import { describe, it, expect, beforeAll } from 'vitest';
+import gql from 'graphql-tag';
+import { ADMIN_USER, queryAsAdmin } from '../../utils/testQuery';
+
+describe('CSV ingestion resolver standard behavior', () => {
+  let singleColumnCsvMapperId = '';
+  let singleColumnCsvFeedIngesterId = '';
+
+  beforeAll(async () => {
+    const SINGLE_COLUMN_CSV_MAPPER = {
+      input: {
+        has_header: false,
+        name: 'Single column CSV mapper',
+        separator: ',',
+        representations: '[{"id":"75c3c21c-0a92-497f-962d-4e6e1a488481","type":"entity","target":{"entity_type":"IPv4-Addr"},"attributes":[{"key":"value","column":{"column_name":"A"},"based_on":null}]}]',
+        skipLineChar: ''
+      }
+    };
+
+    const createSingleColumnCsvMapperQueryResult = await queryAsAdmin({
+      query: gql`
+      mutation createSingleColumnCsvMapper($input: CsvMapperAddInput!) {
+        csvMapperAdd(input: $input) {
+          id
+        }
+      },
+      `,
+      variables: SINGLE_COLUMN_CSV_MAPPER
+    });
+
+    singleColumnCsvMapperId = createSingleColumnCsvMapperQueryResult?.data?.csvMapperAdd?.id;
+  });
+
+  it('should create a CSV feeds ingester', async () => {
+    const CSV_FEED_INGESTER_TO_CREATE = {
+      input: {
+        authentication_type: 'none',
```

Comment:
Do you think you could cover other auth methods in the tests?

Reply:
I will take care of it in a complementary PR